packages/vuetify/src/stylus/components/_data-table.styl

```diff
@@ -22,6 +22,12 @@ v-datatable($material)
     color: rgba($material.fg-color, $material.secondary-text-percent)
     border-top: 1px solid rgba($material.fg-color, $material.divider-percent)
 
+  .v-datatable__expand
+    &-col
```

Comment:
__ and -- separates block/element/modifier, but "expand-col" is just one thing and shouldn't be splitted (i don't think splitting into lines here is necessary, but definitely not this one)

Reply:
Thanks, fixed it :D